### DISCUSSION TITLE
Local Only and not able to use over the Unifi.ui.com?

### DISCUSSION THREAD

**Discussion by ccg-david** [danielfernau/unifi-protect-video-downloader#254] [Q&A]
Simple, yet dumb question. This is only able to be used for a local Protect instance and not one that you can access over the web, correct? 

Or is there a way to use the access to unifi.ui.com and save those events via Docker/Synology as well?

**Comment by danielfernau** [comment]
Hi there! Yes, as of right now the tool is only able to interact with systems on your local network (or via VPN tunnel) - but not via unifi.ui.com. However, that's a good suggestion. If I have a few minutes to spare during the upcoming holidays I'll have a look at it.
